packages/calcite-components/src/components/accordion-item/accordion-item.scss

```diff
@@ -63,38 +77,42 @@
 }
 
 :host .icon {
-  @apply text-color-3
-    relative
+  @apply relative
     m-0
     inline-flex
     duration-150
     ease-in-out;
-  margin-inline-end: var(--calcite-accordion-item-icon-spacing-start);
-  margin-inline-start: var(--calcite-accordion-item-icon-spacing-end);
+  color: var(
+    --calcite-accordion-item-icon-color,
+    var(--calcite-accordion-item-text-color, var(--calcite-color-text-3))
+  );
+  margin-inline-end: var(--calcite-internal-accordion-item-icon-spacing-start);
+  margin-inline-start: var(--calcite-internal-accordion-item-icon-spacing-end);
 }
 
 .icon--start {
   @apply flex items-center;
-  margin-inline-end: var(--calcite-accordion-icon-margin);
+  margin-inline-end: var(--calcite-internal-accordion-item-icon-space-x);
 }
 
 .icon--end {
   @apply flex items-center;
-  margin-inline-end: var(--calcite-accordion-icon-margin);
-  margin-inline-start: var(--calcite-accordion-icon-margin);
+  margin-inline-end: var(--calcite-internal-accordion-item-icon-space-x);
+  margin-inline-start: var(--calcite-internal-accordion-item-icon-space-x);
 }
 
 .header-container {
-  inline-size: 100%;
+  inline-size: var(--calcite-container-size-content-fluid);
 }
 
 .content {
-  padding: var(--calcite-accordion-item-padding);
+  padding: var(--calcite-internal-accordion-item-space-inset);
```

Comment:
Nitpick since this is internal, but shouldn't this be `--calcite-internal-accordion-item-content-space`? 

Applies to other inset space CSS props.